astropy/cosmology/tests/test_cosmology.py

```diff
@@ -1582,6 +1582,46 @@ def test_z_at_value():
             z_at_value(cosmo.angular_diameter_distance, 1500*u.Mpc, zmin=4.)
 
 
+@pytest.mark.skipif('not HAS_SCIPY')
+@pytest.mark.parametrize('method', ['Brent', 'Golden', 'Bounded'])
```

Comment:
Should an issue be raised for future test writing with a custom method?

Reply:
Do you mean raise an issue here somehow or leave a `#TODO:`, or open an issue for a follow-up PR?
As noted above, I don't have a clear idea what custom solver might be used there, or how to test it.
Perhaps one could just take the example from the scipy tutorial, but that would probably mostly test `minimize_scalar`.

Reply:
I was thinking a TODO or issue. But given that this is mostly a scipy behavior... I'm also not sure what a good test would be.

Reply:
Perhaps I'll look in the SciPy tests if they have an example for this.